### ISSUE TITLE
[Feature Request] Renaming Collection

### ISSUE THREAD

**Issue by panoet** [typesense/typesense#344]
## Description

I want to be able to rename my collection.

## Metadata

**Typsense Version**: Cloud

**OS**: -

**Comment by kishorenc** [comment]
👋 @panoet Thanks for creating this feature request. 

For now, have you considered [using an alias](https://typesense.org/docs/0.21.0/api/collection-alias.html#create-or-update-an-alias)? It should be a nice work around. You can decouple the real collection name from the name used to refer to it, so you can keep changing the alias.

**Comment by panoet** [comment]
@kishorenc yes, I already using it. But, I think ability to change our collection name (and scheme, if possible just like firebase) is a basic needs.

**Comment by bakcay** [comment]
Same feature needed actually. Batch import takes huge time and rename feature required for replacing old one.
